Cactooz/reShuffle
# Improve playback buttons

### Tasks completed
- Use SDK for playback options when playing locally and regular fetches otherwise
- Add reshuffle after the end of the playlist, works either by timers or when pressing next on the last track.
- Disable all playback options until the first playlist is played

### Testing
1. Try the playback buttons both when controlling the music locally and remotely from another device
2. Check that the buttons are disabled until the first playlist is played
3. See that the playlist reshuffles at the end of a playlist, both by pressing next and by letting the last song finish.
